client/src/components/AllUsers.js

```diff
@@ -1,129 +1,153 @@
-		setSearch(value);
-	};
+  const handleSearch = (event) => {
+    const { value } = event.target;
+    setSearch(value);
+  };
 
-	function handleDailyFilter() {
-		setFilter("daily");
-	}
+  function handleDailyFilter() {
+    setFilter("daily");
+  }
 
-	function handleWeeklyFilter() {
-		setFilter("weekly");
-	}
+  function handleWeeklyFilter() {
+    setFilter("weekly");
+  }
 
-	function handleMonthlyFilter() {
-		setFilter("monthly");
-	}
+  function handleMonthlyFilter() {
+    setFilter("monthly");
+  }
 
-	return (<div>
-		<div>
-			<NavBar />
-		</div>
-		{trainees.length <=1 ? (<div>
-			<ResultPage filter={setFilter}/>
-		</div>) :(<div>
-			<h1 style={{ color: "yellow", textAlign: "center" }}>Trainees with matching availability</h1>
-			<div className="container">
-				<div className="row">
-					<div className="col-md-6">
-						<input
-							type="text"
-							className="form-control"
-							placeholder="Search"
-							value={search}
-							onChange={handleSearch}
-						/>
-					</div>
-					<div className="col-md-6">
-						<div className="btn-group" role="group" aria-label="Basic example">
-							<button
-								type="button"
-								className="btn btn-outline-info"
-								onClick={handleDailyFilter}
-							>
-								Daily
-							</button>
-							<button
-								type="button"
-								className="btn btn-outline-info"
-								onClick={handleWeeklyFilter}
-							>
-								Weekly
-							</button>
-							<button
-								type="button"
-								className="btn btn-outline-info"
-								onClick={handleMonthlyFilter}
-							>
-								Monthly
-							</button>
-						</div>
-					</div>
-				</div>
-			</div>
-			<div>
-				<span>
-					<div className="col-md-6">
-						<div className="btn-group" role="group" aria-label="Basic example">
-						</div>
-					</div>
-				</span>
-			</div>
-			<div className="container">
-				<div className="row">
-					{trainees.map((trainee) => {
-						return (
-							<div className="col-md-6">
-								<div
-									className="card mb-4 shadow-sm"
-									style={{
-										background: "#2f4d0a",
-										border: "black solid 1px",
-										margin: "2px",
-										padding: "2px",
-									}}
-								>
-								<AvailabilityCards
-										key={trainee.id}
-										trainee={trainee}
-										trainees={trainees}
-										setTrainees={setTrainees}
-									/>
-								</div>
-							</div>
-						);
-					})}
-				</div>
-			</div>
-		</div>)}
-		</div>
-	);
+  return (
+    <div>
+      <div>
+        <NavBar />
+      </div>
+      {trainees.length <= 1 ? (
+        <div>
+          <ResultPage filter={setFilter} />
+        </div>
+      ) : (
+        <div>
+          <h1
+            style={{
+              fontFamily: "sen",
+              color: "black",
+              textAlign: "center",
+              marginTop: "100px",
+              marginBottom: "50px",
+            }}
+          >
+            Trainees with <br /> matching availability
```

Comment:
instead of adding a break eleement in the middle of the sentence this should be acheived with defining the width in css and wrapping the text.
Would be good to see a reuse of the heading component here (which should take a string as a prop)

Reply:
Also this is the wrong heading for this page, will need to update the text.

Reply:
The br is not needed anymore had to remove it, as I had to change the heading name. However, I will definitely use your feedback in future work.